settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
Hmm, that's really long. Bundling both into one option feels wrong, considering those are two different access control mechanisms.

@ares, @iNecas: any opinions on this?